libs/platform/ebpf_platform.h

```diff
@@ -1138,73 +1138,6 @@ extern "C"
         TraceLoggingOpcode(WINEVENT_OPCODE_STOP),                       \
         TraceLoggingString(__FUNCTION__, "==>"));
 
-#define EBPF_RETURN_RESULT(status)                 \
```

Comment:
These macros were sandwiched between the logging ones so I moved them after the logging macros.  IMO, keeping all logging macros together aids in code readability.